src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -70,6 +83,18 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * Map .Entry e = null; // violation, no whitespace before '.' is allowed
+ * Map.Entry f = null; // OK
+ * int foo
+ * ; // OK, linebreak is allowed
+ * Object o = new java.lang.Object();
+ * o . toString(); // violation, whitespace before '.' is not allowed
```

Comment:
This line and line 88 are showing same thing, but their violation comments differ.

Reply:
Instead of showing same case,  please show case of chain calls where each line starts with dot, allowLineBreaks was added to support changed calls that become so popular.